examples/granular/history_test/particles.dat

```diff
@@ -0,0 +1,20 @@
+Python generated LAMMPS data file
+
+3 atoms
+2 atom types
+
+-0.03 0.03 xlo xhi
+-0.02 0.02 ylo yhi
+-0.02 0.02 zlo zhi
+
+Atoms # sphere
+
+1 2 0.0005 2500 0.005 2.7937481665024794E-05 0.0016005372526493155 0 0 0
+2 1 0.01 2500 0.01 0.0 0.0 0 0 0
+3 1 0.01 2500 0.0 0.0 0.0 0 0 0
+
+Velocities
+
+1 0  0 0 0 0 0
+2 0  0 0 0 0 0
+3 0  0 0 0 0 0
```

Comment:
Please do not add files that are missing the final newline character.
Also, if this is a test case, this should not be added to the LAMMPS distribution but rather you make a .tar.gz archive from those files and attach that to the pull request instead.

Reply:
Thanks! I've removed the test case from the examples and attached it as a zip file.